### ISSUE TITLE
Sharing side-menu between components

### ISSUE THREAD

**Issue by Nosherwan** [Kureev/react-native-side-menu#101]
I am creating an application where links within the side-menu open separate page components. Each one of these pages have their own instances of navigators. In each of these pages the sideMenu is being rendered like so:
```javascript
    render() {
        var Menu = require('./AppMenu');
        var SideMenu = require('react-native-side-menu');
        var menu = <Menu
            navigator={this.props.navigator}/>;

        var navigationBar = <Navigator.NavigationBar
            routeMapper={NavigationBarRouteMapper}
            />;
        return (
            <SideMenu menu={menu} animation='slide' openMenuOffset='280' touchToClose={true} disableGestures={true}>
                <Navigator
                    ref="refNavigator"
                    initialRoute={{id: 'subPage'}}
                    navigator={this.props.navigator}
                    renderScene={this.renderScene()}
                    navigationBar={navigationBar}
                    changeNavBarStyle={this.changeNavBarStyle.bind(this)}
                    />
            </SideMenu>
        );
    }

 renderScene() {
        var self = this;

        return function (route, nav) {
            switch (route.id) {
                case 'subPage':
                    return (
                        <View>
                            <ListView
                                dataSource={self.state.dataSource}
                                renderRow={self._renderRow.bind(self)}
                                renderHeader={self._renderSectionHeader.bind(self)}
                                automaticallyAdjustContentInsets={true}
                                >
                            </ListView>
                        </View>
                    );
                case 'customPage':
                    return (
                        <CustomPage navigator={nav}/>
                    );
```
This way there is a weard animating effect going from one page to the other.
This situation is not working well as ideally the sidemenu should be shared among the navigators. Can you suggest a way in which we can share the sidemenu among multiple navigators or any other approach?

**Comment by Kureev** [comment]
Solved in `0.14.0`. Check the [release notes](https://github.com/Kureev/react-native-side-menu/releases/tag/v0.14.0).

**Comment by JayGajjar** [comment]
@Kureev In release notes you have mention about example. But i cannot find such example in examples/Basic

Can you please show me how i can call toggle function of Basic.js on button click of another Component let say it Home.js

UseCase is simple : I have a button on Home.js and on click of it i need to open slider menu.
